src/components/header.js

```diff
@@ -21,40 +18,39 @@ const Header = () => {
       }}
     >
       <Container sx={{ variant: `layout.container.header` }}>
-        <div sx={{ width: `max-content`, whiteSpace: `nowrap`, mr: `2` }}>
+        <div sx={{ width: `max-content`, whiteSpace: `nowrap` }}>
           <Link to="/">
             <span sx={{ display: [`none`, `initial`] }}>Recipe Palette </span>
             <span role="img" aria-label="palette">
               🎨
             </span>
           </Link>
         </div>
-        <div sx={{ width: `100%`, ml: [`2`, `3`], mr: [`0`, `3`] }}>
+        <div sx={{ width: `100%` }}>
           <SearchBar />
         </div>
         <div
           sx={{
-            '*+*': { marginLeft: `3` },
             display: [`none`, `flex`],
             alignItems: `center`,
             width: `max-content`,
             whiteSpace: `nowrap`,
-            ml: `2`,
           }}
         >
-          <Link sx={{ variant: `button.link` }} to="/my-recipes">
+          <Link sx={{ variant: `buttons.link` }} to="/my-recipes">
             My Recipes
           </Link>
-          <MdAccountBox
-            size={50}
-            sx={{
-              display: `inline-flex`,
-              cursor: `pointer`,
-              color: lighten(`primary`, 0.25),
-              '&:hover': { color: lighten(`primary`, 0.1) },
-            }}
-            onClick={() => (isAuthenticated() ? customLogout() : login())}
-          />
+        </div>
+        <div>
```

Comment:
Do we want the profile box to show up in mobile in the top righthand corner? We have it in the nav in the bottom, too